servant-client/test/Servant/ClientSpec.hs

```diff
@@ -30,7 +30,7 @@ module Servant.ClientSpec where
 import           Control.Applicative        ((<$>))
 #endif
 import           Control.Arrow              (left)
-import           Control.Concurrent         (forkIO, killThread, ThreadId)
+import           Control.Concurrent         (ThreadId, forkIO, killThread)
```

Comment:
My guess would be that it'd be hard to come to a consensus, which style and tool to use for code-formatting. For example, I personally use https://github.com/zalora/normalize-imports. And I also don't think it's worth the effort, at least not at this point.

I *do* think that we shouldn't have formatting changes scattered around in unrelated PRs and commits like this. In my opinion, if you touch a piece of code, please, go nuts and reformat at will. If you don't, then don't.

What I usually do is to run my code-formatting tools during development and they might change formatting where I don't want them to. Then I commit changes selectively with `git add -p`, while not staging the formatting changes in parts of the code that I didn't really touch. Afterwards I run `git checkout -p`, to revert the unwanted changes the tools introduced.

I think another good workflow that I heard about is this: You configure your editor to allow you to format only regions of the code. Like the selected code block, or the enclosing expression.